src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -323,15 +325,15 @@ public Builder addStdErrLineListener(ProcessOutputLineListener stdErrLineListene
      * The client listener of the process exit with code.
      */
     public Builder exitListener(ProcessExitListener exitListener) {
```

Comment:
I think we would make it plural, but no 'add' prefix. Just builder.exitListners(...)

In any case, if you want to keep the surface the same, you should first clear the exitListeners list in this method. Otherwise, the method no longer works like a setter.